packages/react-events/src/Focus.js

```diff
@@ -39,6 +39,8 @@ type FocusEvent = {|
   timeStamp: number,
 |};
 
+const isMac = /^Mac/.test(navigator.platform);
```

Comment:
Needs to be safe to evaluate this on the server: https://github.com/facebook/react/blob/0f7cc2b/packages/react-events/src/Press.js#L104-L107